docs/source/build-options.rst

```diff
@@ -12,15 +12,15 @@ Build and configuration
 Configuration
 -------------
 
-``xtensor`` can be configured via macros which must be defined *before* including
+`xtensor` can be configured via macros which must be defined *before* including
 any of its headers. This can be achieved the following ways:
 
 - either define them in the CMakeLists of your project, with ``target_compile_definitions``
   cmake command.
 - or create a header where you define all the macros you want and then include the headers you
-  need. Then include this header whenever you need ``xtensor`` in your project.
+  need. Then include this header whenever you need `xtensor` in your project.
```

Comment:
As far as I know single back quoted text is nothing in restructured text. I think it is only used as a command, such as ``:cpp:fun:`...` ``. 

(There are several occurrences of this)

Reply:
It renders as italic, but it seems to be nonstandard.

When referring to xtensor as the library, there were both usage of ``` ``xtensor`` ``` and ``` `xtensor` ```.
I believe the italic version is less confusing, so I started changing them (but not exhaustively).

Reply:
Why not privilege `*xtensor*`